playbooks/loki.yaml

```
@@ -0,0 +1,44 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Configure loki.
+---
+- name: Configure loki
+  hosts: loki_servers
+  become: true
+  tasks:
+    - name: Install loki
+      ansible.builtin.package:
+        name: loki
+        state: present
+
+    - name: Configure loki
+      ansible.builtin.template:
+        src: files/loki/config.yml.j2
+        dest: /etc/loki/config.yml
+        owner: root
+        group: root
+        mode: '0644'
```

this should trigger a restart when the config is changed
```suggestion
        mode: '0644'
      notify: _tina_restart_loki
```